lib/msal-core/src/UserAgentApplication.ts

```diff
@@ -1168,11 +943,13 @@ export class UserAgentApplication {
    * Used to redirect the browser to the STS authorization endpoint
    * @param {string} urlNavigate - URL of the authorization endpoint
    */
-  private promptUser(urlNavigate: string) {
+  private navigateWindow(urlNavigate: string, popupWindow?: Window) {
     // Navigate if valid URL
     if (urlNavigate && !Utils.isEmpty(urlNavigate)) {
-      this.logger.infoPii("Navigate to:" + urlNavigate);
-      window.location.replace(urlNavigate);
+      let navigateWindow: Window = popupWindow ? popupWindow : window;
+      let logMessage: string = popupWindow ? "Navigated Popup window to:" + urlNavigate : "Navigate to:" + urlNavigate;
```

Comment:
Rereading this noticing that `popupWindow` will never be falsey, so the second log message condition will never happen.

Reply:
I'm not sure what you mean, popupWindow will be falsey if navigateWindow is called in loginRedirect or acquireTokenRedirect, since we don't pass a window object.

Reply:
Ah whoops misread it, disregard.